docs/data/base/components/slider/DiscreteSlider.tsx

```diff
@@ -109,7 +109,7 @@ const StyledSlider = styled(SliderUnstyled)(
     background-color: #fff;
   }
 
-  & .${sliderUnstyledClasses.valueLabel} {
+  & .valueLabel {
```

Comment:
Can we improve the aligning, seems to be a bit off:
![image](https://user-images.githubusercontent.com/4512430/212885066-0935d577-7e09-4289-b720-77c6ee969454.png)

(can be in a separate PR, as it was like this before)

Reply:
Fixed in commit https://github.com/mui/material-ui/pull/35805/commits/0acf55c379f5635f0056eeae9fff10e325ddb4a9.

Reply:
How about we do something like this instead: https://codesandbox.io/s/nifty-black-k1qlbz?file=/demo.tsx Using -35% doesn't work with the three digits numbers.

Reply:
Okay, it works well. I changed in commit https://github.com/mui/material-ui/pull/35805/commits/ed47db22533bead15a20e8a1b717f17271572b2e. I am not sure why you used `display: inline-flex` instead of `display: flex` since there is no other element on left or right.

Reply:
Looks good, thanks for the update.